### PR TITLE
Added Features

### PR DESCRIPTION
**Pull request by nishasen**
Features contains the different states used across the app, it is done using redux.

### CODE REVIEW

File: social/src/Routepath/Routepath.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      </Route>
+      <Route element={<RequireAuth />}>
+        <Route path="/home" element={<Home />}/>
+        <Route path="/explore" element={<Explore />}/>
+        <Route path="/notification" element={<Notification />}/>
+        <Route path="/bookmark" element={<Bookmark />}/>
+        <Route path="/profile" element={<Profile />}/>
+        <Route element={<ProfileNavigate />}>
+          <Route path="/user/:username" element={<User />}/>
+        </Route>
```
Comment (sadathshariff) [inline social/src/Routepath/Routepath.jsx:31]: Appreciation, Nice way of handling the Profile page of other users

**Review by sadathshariff — APPROVED** [review]
Looks Good, learned about handling the Profile Page of users using ProfileNavigate

File: social/src/Features/Auth/AuthSlice.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    setOtherUser: (state, action) => {
+      state.otherUser = action?.payload?.data;
+    }
+  },
+  extraReducers: {
+    [getUser.pending]: (state) => {
+      state.loading = true;
+    },
+    [getUser.fulfilled]: (state, action) => {
```
Comment (srejitk) [inline social/src/Features/Auth/AuthSlice.js:40]: You can use builder here for a much cleaner code. Have a look into it.

**Review by srejitk — APPROVED** [review]
Clean readable code. Good work hanging the post sorting and theme slice!

File: social/src/Features/Auth/AuthSlice.js (hunk 2 above)
Comment (nishasen) [reply social/src/Features/Auth/AuthSlice.js:40]: Okay sure

**Pull request merged**